blog/authors.yml

```diff
@@ -52,4 +52,9 @@ Trilok Geer:
 KubeEdge SIG Release:
   name: KubeEdge SIG Release
   url: https://github.com/kubeedge-bot
-  image_url: https://avatars.githubusercontent.com/u/48982446?v=4
\ No newline at end of file
+  image_url: https://avatars.githubusercontent.com/u/48982446?v=4
+
+Tomoya Fujita:
+  name: Tomoya Fujita
+  url: https://github.com/fujitatomoya
+  image_url: https://avatars.githubusercontent.com/u/43395114?v=4
```

Comment:
Is this supposed to be merged before gets linked? if that is true, i can submit only this file with another PR.